pyproject.toml

```diff
@@ -43,7 +43,8 @@ dependencies = [
     "magicattr~=0.1.6",
     "litellm",
     "diskcache",
-    "json-repair"
+    "json-repair",
+    "tenacity>=8.2.3",
```

Comment:
LiteLLM uses `tenacity==8.2.3` for testing: https://github.com/BerriAI/litellm/blob/b73039b28337c04ac87eea8e0be11186f142f38f/requirements.txt#L46.

I've confirmed that 8.2.3 *and* 9.0.0 appear to work just fine via the following script using an Azure OpenAI + Databricks endpoint with an aggressive rate limit of 1QPM:

```
import litellm
import uuid


litellm.set_verbose=True
for i in range(100):
    print(litellm.completion(
        model="databricks/coreyazureopenai",
        messages=[{"role": "user", "content": "You are a helpful assistant." + uuid.uuid4().hex}],
        retry_strategy="exponential_backoff_retry",
        num_retries=10,
    ))
```

Accordingly, `>=8.2.3` appears appropriate